update UI documentation

sloanej opened the pull request:
This PR includes the updated UI templates in "Documentation/UI Template.fig". New screen templates are added for the profile screen and discussion-related screens. Additionally, the background images and icons used in the templates are stored in "EdxMobile/assets", and the icon paths are added to Icon.tsx.

jonmahoney15 commented on Documentation/Assets/vector icons/angle-down-solid.svg:
all of the icons can be moved to the asset/icon folder in the EdxMobile instead of here

sloanej replied:
Ok. I moved them. The commit now reflects this change.

jonmahoney15 commented:
Okay this looks good, however the .DS_Store need to be removed, if you rebase your main branch, itll add the gitignore and should remove them

Commit:
update ui templates
This commit includes the updated UI templates in "Documentation/UI Template.fig". New screen templates are added for the profile screen and discussion-related screens. Additionally, the background images and icons used in the templates are stored in "EdxMobile/assets", and the icon paths are added to Icon.tsx.

sloanej commented:
> Okay this looks good, however the .DS_Store need to be removed, if you rebase your main branch, itll add the gitignore and should remove them

Okay.  I think we're good now.
